### PR TITLE
It lang updated

### PR DESCRIPTION
**Pull request by ITKewai**

### CODE REVIEW

File: language/it.json [hunk 1]
```diff
@@ -33,35 +33,35 @@
...
     "badidea": "Non farlo, per piacere.",
-    "viewbanlist": "View Banlist",
+    "viewbanlist": "Visualizza Banlist",
     "unbanplayer": "Rimuovi ban per il giocatore",
     "unbanplayerguide": "~r~~h~NOTA:~h~~w~  Premendo Conferma Rimuoverai il Ban di questo giocatore.",
     "banlistshowtype": "~h~Banlist:~h~ Mostra tipo",
     "unbanreasons": "Ragioni",
     "unbanlicenses": "Licenze",
-    "banlistshowtypeguide": "Cambia tra Identificatori e Ragioni per il ban Nel menu 'Rimuovi Ban per il giocatore'.\nRichiede una riapertura del menu.",
+    "banlistshowtypeguide": "Cambia tra Licenza e Ragioni per il ban Nel menu 'Rimuovi Ban per il giocatore'.\nRichiede una riapertura del menu.",
```
Comment (Blumlaut) [inline language/it.json:64]: Why do some of these specifically mention a license?

File: language/it.json [hunk 2]
```diff
@@ -76,25 +76,25 @@
     "chatsuggestionban": "bandisci un giocatore",
     "chatsuggestionkick": "espelli un giocatore",
     "chatsuggestionspectate": "spetta un giocatore",
-    "chatsuggestionunban": "bandisci un identificatore",
+    "chatsuggestionunban": "bandisci una licenza",
```
Comment (Blumlaut) [inline language/it.json:79]: same with this one, whats wrong with it?

File: language/it.json [hunk 3]
```diff
@@ -105,71 +105,71 @@
...
+    "cleancarsguide": "Rimuove i veicoli vicino a te.",
+    "cleaningcar": "Rimuovendo veicolo %s",
+    "cleanpeds": "Rimuovi peds",
+    "cleanpedsguide": "Sto rimuovendo i peds.",
+    "cleaningped": "Rimuovendo Ped %s",
+    "cleanprops": "Rimuovi Props",
+    "cleanpropsguide": "Rimuove gli oggetti vicini spawnati dai cheater.",
+    "cleaningprop": "Rimuovendo oggetto %s",
+    "finishedcleaning": "Ho finito di rimuovere %s",
+    "admincleanedup": "**%s** Ha cancellato tutti  **%s** in their area."
```
Comment (Blumlaut) [inline language/it.json:174]: this translation is incomplete

**Review by Blumlaut — CHANGES_REQUESTED** [review]

File: language/it.json (hunk 2 above)
Comment (ITKewai) [reply language/it.json:79]: Actually license sound better in Italian than indentifier, but the meaning is the same

**Comment by Blumlaut** [comment]
Thanks!

**Pull request merged**